docs/topics/rn-known-issues.adoc

```diff
@@ -18,21 +18,17 @@ For a complete list of all known issues, see the list of link:https://issues.red
 
 |link:https://issues.redhat.com/browse/WINDUP-3138[WINDUP-3138]
 |CodeReady Workspace plugin
-|The plugin does not work with CodeReady Workspace 2.10.
+|The plugin works with CodeReady Workspace 2.09, but not version 2.10.
 
-|link:https://issues.redhat.com/browse/WINDUP-3019[WINDUP-3019]
-|CLI
-|Bash completion feature displays an `mta-cli not found` error.
+|link:https://issues.redhat.com/browse/WINDUP-3165[WINDUP-3165]
+|Web console
+|MTA analysis fails on OpenShift Container Platform when custom labels are uploaded and advanced options are enabled.
 
-|link:https://issues.redhat.com/browse/WINDUP-3104[WINDUP-3104]
-|Web console on MacOS
-|After you run an analysis, if you click a report link, a "Not found" page is displayed.
+|link:https://issues.redhat.com/browse/WINDUP-3160[WINDUP-3160]
+|Web console
+|MTA analysis does not run on OpenShift Container Platform if applications are deleted.
```

Comment:
This summary is not quite correct. It's not a case of the analysis not running if applications are deleted. What happens is this: If you run an analysis and let it complete, then delete an application and run the analysis again, the analysis hangs. What that indicates is probably MTA is relying on an OpenShift inventory, which is not updating properly. So the analysis configuration is not updating and it is trying to find the deleted app.

This is closer to the meaning. Rewrite as you see fit: 

If you run an analysis on OpenShift Container Platform on multiple applications and subsequently delete an application and rerun the analysis, the analysis hangs.